plugin.py

```diff
@@ -30,61 +44,54 @@ def is_allowed_to_start(
-                "getDocumentSelectionRequest": False,
-            },
-            "documentHighlight": True,
-            "documentLink": False,
-            "codeLens": False,
-            "semanticTokens": False,
-            "schemaRequestService": False,
-            "diagnostics": True
-        })
-        configuration.init_options.set('documentFeatures', {
-            "selectionRange": True,
-            "foldingRange": True,
-            "linkedEditingRange": False,
-            "documentSymbol": True,
-            "documentColor": True,
-            "documentFormatting": {
-                "defaultPrintWidth": 90
-            }
-        })
+        configuration.init_options.set('languageFeatures', get_language_features(configuration, is_main_server=True))
+        configuration.init_options.set('documentFeatures', document_features)
         if configuration.init_options.get('typescript.serverPath'):
-            return # don't find the `typescript.serverPath` if it was set explicitly in LSP-volar.sublime-settings
-        typescript_path = LspVolarPlugin.find_typescript_path(workspace_folders[0].path)
+            return  # don't find the `typescript.serverPath` if it was set explicitly in LSP-volar.sublime-settings
+        typescript_path = find_typescript_path(LspVolarPlugin, workspace_folders[0].path)
         configuration.init_options.set('typescript.serverPath', typescript_path)
 
+
+class LspVolarSecondServer(LspVolarPlugin):
+    @classmethod
+    def get_displayed_name(cls) -> str:
+        return 'LSP-volar(second server)'
+
     @classmethod
-    def find_typescript_path(cls, current_folder: str) -> str:
-        server_directory_path = cls._server_directory_path()
-        resolve_module_script = os.path.join(server_directory_path, 'resolve_module.js')
-        find_ts_server_command =  [cls._node_bin(), resolve_module_script, current_folder]
-        startupinfo = None
-        # Prevent cmd.exe popup on Windows.
-        if sublime.platform() == "windows":
-            startupinfo = subprocess.STARTUPINFO()
-            startupinfo.dwFlags |= (
-                subprocess.SW_HIDE | subprocess.STARTF_USESHOWWINDOW
-            )
-        workspace_ts_path = subprocess.check_output(find_ts_server_command, universal_newlines=True, startupinfo=startupinfo)
-        bundled_ts_path = os.path.join(server_directory_path, 'node_modules', 'typescript', 'lib', 'tsserverlibrary.js')
-        return workspace_ts_path or bundled_ts_path
+    def is_allowed_to_start(
+        cls,
+        window: sublime.Window,
+        initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        configuration: Optional[ClientConfig] = None
+    ):
+        if not workspace_folders or not configuration:
+            return
+        use_second_server = configuration.settings.get('volar.vueserver.useSecondServer')
+        if not use_second_server:
+            return "Not enabled"
```

Comment:
Currently if `volar.vueserver.useSecondServer` is false, 
I will see an error message `cannot start LSP-volar(second server): Not enable` in the status bar.

Ideally I would not like to display any error message to the user if the second server is not enabled,
is there a way to hide that message?